partiql-planner/build.gradle.kts

```diff
@@ -24,17 +24,13 @@ plugins {
 }
 
 dependencies {
+    api(project(":partiql-ast"))
     api(project(":partiql-plan"))
     api(project(":partiql-types"))
-    implementation(project(":partiql-ast"))
     implementation(Deps.dotlin)
     implementation(Deps.ionElement)
     // Test
     testImplementation(project(":partiql-parser"))
-    testImplementation(project(":plugins:partiql-local"))
-    testImplementation(project(":plugins:partiql-memory"))
-    // Test Fixtures
-    testFixturesImplementation(project(":partiql-spi"))
```

Comment:
partiql-planner no long depends on spi which means we cannot use any of the plugins.